autogen/docs/headings.html.mustache

```diff
@@ -69,7 +69,6 @@
   <span class="block offwhitetext bold nowrap">{{date}}</span>
 </div>
 <div style="margin-bottom: 0em; margin-top: .25em;">
-  <span class="heading-title xsmallfont">Release Notes</span>
```

Comment:
It looked like a section heading for the four items beneath it (I recorded the reason in the commit message). "What's new?" can be changed to "Released Notes", of course, but the current position & look of "Release Notes" is *very* confusing.